src/app/component/test-chat/test-chat.component.html

```diff
@@ -4,36 +4,37 @@
     <div class="input-group-prepend">
       <span class="input-group-text" id="inputGroup-sizing-sm">Patient Id</span>
     </div>
-    <input [(ngModel)]="data.patientId" type="text" class="form-control" />
+    <input id="formControl1" [(ngModel)]="data.patientId" type="text" class="form-control" />
```

Comment:
@VishalMajumdar09 instead of formControl1 can we use form-control-patientId like this?